libs/message-coder/src/uint16_coder.ts

```diff
@@ -11,11 +11,25 @@ import {
 } from './types';
 import { UintCoder } from './uint_coder';
 
+interface Uint16CoderOptions {
+  littleEndian: boolean;
+}
+
 /**
  * Coder for a uint16, aka a 16-bit unsigned integer. Uses little-endian byte
  * order.
  */
 export class Uint16Coder extends UintCoder {
+  private readonly littleEndian: boolean;
+
+  constructor(
+    enumeration?: unknown,
+    options: Uint16CoderOptions = { littleEndian: true }
```

Comment:
This is equivalent for now, but if we add additional properties it allows specifying defaults for one at a time rather than all or nothing.
```suggestion
    { littleEndian = true }: Partial<Uint16CoderOptions> = {}
```